actors/abi/sector.go

```diff
@@ -203,6 +205,14 @@ func (p RegisteredSealProof) RegisteredWindowPoStProof() (RegisteredPoStProof, e
 	}
 }
 
+// SectorMaximumLifetime is the maximum duration a sector sealed with this proof may exist between activation and expiration
+func (p RegisteredSealProof) SectorMaximumLifetime() ChainEpoch {
+	// currently all proofs set to around 5 years
+	epochsPerYear := 1_262_277
```

Comment:
Use the existing `builtin.EpochsInYear` which will change appropriately if epoch duration changes.

Also I can clarify the comment for you: "For all Stacked DRG sectors, the max is 5 years".

Reply:
I would love to, but sector.go is in `abi`, so referencing `builtin` would be circular.

Reply:
Gah, ok. We should probably move this logic out of `abi` then, so it returns to just types.